rna/lr_system.py

```diff
@@ -0,0 +1,100 @@
+import numpy as np
+
+from sklearn.neural_network import MLPClassifier
+
+from lir import KDECalibrator
+from rna.analytics import get_mixture_columns_for_class
+
+class MarginalClassifier():
+
+    def __init__(self, random_state=0, classifier=MLPClassifier,
+                 calibrator=KDECalibrator, MAX_LR=10):
+        self._classifier = classifier(random_state=random_state)
+        self._calibrator = calibrator
+        self._calibrators_per_target_class = {}
+        self.MAX_LR = MAX_LR
+
+    def fit_classifier(self, X, y):
+        self._classifier.fit(X, y)
+
+    def fit_calibration(self, X, y_nhot, target_classes):
+        """
+        Makes calibrated model for each target class
+        """
+        lrs_per_target_class = self.predict_lrs(X, target_classes)
+
+        for i, target_class in enumerate(target_classes):
+            calibrator = self._calibrator()
+            loglrs = np.log10(lrs_per_target_class[:, i])
```

Comment:
gebruikelijker is geloof ik om te calibreren op de kansen. Kunnen we die toevoegen aan het lijstje kalibratiestappen om te testen? :)

Reply:
Zal ik toevoegen